rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -118,6 +119,7 @@ def _parse_airdrops(database: 'DBHandler', airdrops_data: dict[str, Any]) -> dic
                 url=airdrop_data['url'],
                 name=airdrop_data['name'],
                 icon=airdrop_data['icon'],
+                icon_url=f"{AIRDROPS_REPO_BASE}/{airdrop_data['icon_path']}" if 'icon_path' in airdrop_data else None,  # noqa: E501
```

Comment:
you name the attribute icon_url but in the json is icon_path. I believe it should be icon_url in all the places

Reply:
this was because in the metadata, we are returning the path, and in the api response we are returning the url

Reply:
I don't understand here why the difference between icon path and icon url